README.adoc

```diff
@@ -249,14 +249,14 @@ tracing of particular methods. You can also inject a custom `Tracer` object to c
 === Enabling distributed tracing without code instrumentation
 
 Because tracing of all JAX-RS methods is enabled by default, you need only to enable MicroProfile
-OpenTracing and the Zipkin user feature in the [hotspot=7-8 file=0]`server.xml` file to see some basic traces in Zipkin.
+OpenTracing and the Zipkin user feature in the [hotspot file=0]`server.xml` file to see some basic traces in Zipkin.
```

Comment:
this hotspot should be highlighting like this:
![image](https://user-images.githubusercontent.com/46586499/66955261-fff6bc00-f02f-11e9-8090-df47c7c95b91.png)

Reply:
@DannyS03 this is not fixed

Reply:
Updated.